src/components/sidebar/MarkerSidebar.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+
+import explicitConnect from '../../utils/connect';
+import { selectedThreadSelectors } from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import MarkerTooltipContents from '../shared/MarkerTooltipContents';
+
+import type {
+  ConnectedProps,
+  ExplicitConnectOptions,
+} from '../../utils/connect';
+import type { ThreadIndex } from '../../types/profile';
+import type { TracingMarker } from '../../types/profile-derived';
+
+type StateProps = {|
+  +selectedThreadIndex: ThreadIndex,
+  +marker: TracingMarker,
+|};
+
+type Props = ConnectedProps<{||}, StateProps, {||}>;
+
+class MarkerSidebar extends React.PureComponent<Props> {
+  render() {
+    const { marker, selectedThreadIndex } = this.props;
+
+    if (marker === null || marker === undefined) {
+      return (
+        <div className="sidebar sidebar-calltree">
+          Select a marker to display some information about it.
+        </div>
+      );
+    }
+
+    return (
+      <aside className="sidebar sidebar-calltree">
+        <div className="sidebar-contents-wrapper">
```

Comment:
The current style for this class isn't very good in our case here (it has a `display: grid` that we don't need here, and makes things look weird). Maybe change the existing `.sidebar-content-wrapper` style to use `.sidebar-calltree .sidebar-content-wrapper` ? Just keep the `line-height` for `.sidebar-content-wrapper`.